slackbot/manager.py

```diff
@@ -72,3 +73,6 @@ def get_plugins(self, category, text):
 
         if not has_matching_plugin:
             yield None, None
+
+    def get_idle_plugins(self):
+        yield from self.idle_commands
```

Comment:
@csaftoiu: Not compatible with Python2.7. Can be changed to the following -
`for c in self.idle_commands:
    yield c`